incubator/exporter-otlp.spec/src/main/scripts/io/aklivity/zilla/specs/exporter/otlp/network/event/client.rpt

```diff
@@ -0,0 +1,19 @@
+#
+# Copyright 2021-2023 Aklivity Inc
+#
+# Licensed under the Aklivity Community License (the "License"); you may not use
+# this file except in compliance with the License.  You may obtain a copy of the
+# License at
+#
+#   https://www.aklivity.io/aklivity-community-license/
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+# WARRANTIES OF ANY KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations under the License.
+#
+
+connect "zilla://streams/net0"
+    option zilla:window 8192
+
+connected
```

Comment:
Is this needed or is there a spec from engine that we should be using instead?
Also noticed there is no server.rpt for this client.rpt.

Reply:
yep, we can get this from the engine, fixed
also, I'm done adding the missing script pairs, should be good now